### DISCUSSION TITLE
are broken websites ever going to be fixed?

### DISCUSSION THREAD

**Discussion by lica98** [darkreader/darkreader#11846] [Q&A]
I've reported several websites a long time ago but literally not a single one was ever fixed, do issues ever get addressed here?
for example:
https://github.com/darkreader/darkreader/issues/10787
https://github.com/darkreader/darkreader/issues/10618
https://github.com/darkreader/darkreader/issues/10967

**Comment by gshpychka** [comment]
Dark theme detection reports also seem to be ignored: https://github.com/darkreader/darkreader/discussions/9016
